src/autoload.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+use interop\container\containerinterface as InteropContainerInterface;
+use interop\container\exception\containerexception as InteropContainerException;
+use interop\container\exception\notfoundexception as InteropNotFoundException;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\ContainerInterface;
+use Psr\Container\NotFoundExceptionInterface;
+
+class_alias(ContainerInterface::class, InteropContainerInterface::class);
```

Comment:
This block kinda needs to be conditional, IMO.

If `InteropContainerInterface` is not defined, we can certainly declare it ourselves, but on a system that has `container-interop/container-interop` installed, this feels like we're stepping on some other maintainer's toes.

Reply:
it cannot be defined as we replace it while conflicting with older versions. thus, no need to add any conditon. and if we add psr/container v2, we still should alias the interface until we release a new major, not?